tests/unit/algorithms/detection/tiling/test_tiling_tile_classifier.py

```diff
@@ -81,9 +81,18 @@ def test_openvino_sync(self, mocker):
         mocked_model.return_value = mocker.MagicMock(spec=MaskRCNNModel, model_adapter=adapter_mock)
         params = DetectionConfig(header=self.hyper_parameters.header)
         ov_mask_inferencer = OpenVINOMaskInferencer(params, self.label_schema, "")
-        ov_mask_inferencer.model = mocked_model
+        original_shape = (self.dataset[0].media.width, self.dataset[0].media.height, 3)
         ov_mask_inferencer.model.resize_mask = False
-        ov_mask_inferencer.model.preprocess.return_value = ({"foo": "bar"}, {"baz": "qux"})
+        ov_mask_inferencer.model.preprocess.return_value = (
+            {"foo": "bar"},
+            {"baz": "qux", "original_shape": original_shape},
+        )
+        ov_mask_inferencer.model.postprocess.return_value = (
+            np.array([], dtype=np.float32),
+            np.array([], dtype=np.uint32),
+            np.zeros((0, 4), dtype=np.float32),
+            [],
+        )
         ov_inferencer = OpenVINOTileClassifierWrapper(
```

Comment:
This test requires update to be compatible with the new version of the MAPI. @GalyaZalesskaya could you have a look? Probably, we have to mock more methods / functions 
tests/unit/algorithms/detection/tiling/test_tiling_tile_classifier.py:96
```
self = <MagicMock name='create_model()' spec='MaskRCNNModel' id='139921047417328'>
name = 'inference_adapter'

    def __getattr__(self, name):
        if name in {'_mock_methods', '_mock_unsafe'}:
            raise AttributeError(name)
        elif self._mock_methods is not None:
            if name not in self._mock_methods or name in _all_magics:
>               raise AttributeError("Mock object has no attribute %r" % name)
E               AttributeError: Mock object has no attribute 'inference_adapter'

name       = 'inference_adapter'
self       = <MagicMock name='create_model()' spec='MaskRCNNModel' id='139921047417328'>
```

Reply:
Sure, I'll take a look